### ISSUE TITLE
Provide PyPI Package

### ISSUE THREAD

**Issue by erayerdin** [AWConant/jikanpy#18]
It is possible to install Python packages via Git, but the official hosting for Python projects is PyPI.

This is not something I can do with PR though. You might want to check some documentations.

Travis also supports automatic PyPI artifact uploads if builds pass and you provide tag via Git. You can [check this Travis configuration](https://github.com/erayerdin/pyairmore/blob/master/.travis.yml#L63) to automatize it with encrypting your password.

Or you can do it via good-old manual way as well.

This issue can wait by the way.

**Comment by erayerdin** [comment]
By the way, this process is probably the hardest because it needs getting encrypted PyPI password from Travis (it uses Ruby), adapting the publish automation to Travis and a couple of steps. I have already done this before.

If @AWConant allows me, I can host `jikanpy` package on PyPI. This doesn't have any major drawbacks. Then one can use git tags to automatically publish new releases on PyPI. For instance, you have just committed and Travis passes without any problem. Then you can simply do:

```
# let's assume the version is 0.1.0
git tag -a "v0.1.0"
git commit origin v0.1.0
```

This will automatically publish the package on PyPI as `0.1.0` version.

For [me](https://pypi.org/user/erayerdin/) to host it on PyPI, it will be easier for me to create a PR for this. If, somehow, you want me to remove the package from PyPI in the future, you can contact me again.

You can get more information about git tags from [here](https://dev.to/neshaz/a-tutorial-for-tagging-releases-in-git-147e).

**Comment by AWConant** [comment]
That sounds good to me; go for it. I don't know much about PyPI.

**Comment by abhinavk99** [comment]
Done in v3.4.0 (commit b7708ec)

https://pypi.org/project/jikanpy/
